app/views/vat/claimVATInterstitialPage.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
```

Comment:
Format and attachscreenshots in Jira ticket before change (staging) and after change (local) . ALl looks good to me.

Reply:
sure, done now, thanks